.github/workflows/build.yml

```
@@ -39,6 +39,31 @@ on:
         required: false
         type: string
         default: 'v1'
+      USE_DYNAMIC_IMAGE_TAG:
+        description: 'If this field is set to true, the image tag configured in the config.yml file will be skipped, and a dinamic generated image tag will be used instead.'
+        required: false
+        type: boolean
+        default: false
+      UPDATE_REMOTE_CONFIG:
```

What do you think of something a little more specific like:
```suggestion
      UPDATE_IMAGE_TAG_IN_REPO:
```
?